benefits/static/css/styles.css

```diff
@@ -75,6 +75,11 @@ main .main-row {
   min-height: calc(100vh - 120px);
 }
 
+main .main-row .col-lg-6.image {
+  background: 48% 75% url("/static/img/ridertappingbankcard.png") no-repeat;
```

Comment:
Since this path moved to the stylesheet, can we please clean up the [`core.viewmodels.Page`](https://github.com/cal-itp/benefits/blob/dev/benefits/core/viewmodels.py#L144), which also sets a path to this image.

Reply:
Removed